code/modules/projectiles/ammo_datums.dm

```diff
@@ -1695,58 +1695,57 @@ datum/ammo/bullet/revolver/tp44
 	smoke_system.start()
 
 /datum/ammo/xeno/toxin/upgrade1
-	smoke_strength = 0.55
-	reagent_transfer_amount = 7
+	smoke_strength = 0.6
+	reagent_transfer_amount = 8
 
 /datum/ammo/xeno/toxin/upgrade2
-	smoke_strength = 0.6
-	reagent_transfer_amount = 7.5
+	smoke_strength = 0.7
+	reagent_transfer_amount = 8.5
 
 /datum/ammo/xeno/toxin/upgrade3
-	smoke_strength = 0.65
-	reagent_transfer_amount = 8
-
+	smoke_strength = 0.8
+	reagent_transfer_amount = 9
 
 /datum/ammo/xeno/toxin/medium //Queen
 	name = "neurotoxic spatter"
 	added_spit_delay = 10
-	spit_cost = 75
-	damage = 35
-	smoke_strength = 0.6
-	reagent_transfer_amount = 7.5
+	spit_cost = 85
+	damage = 45
+	smoke_strength = 0.7
+	reagent_transfer_amount = 8
 
 /datum/ammo/xeno/toxin/medium/upgrade1
-	smoke_strength = 0.65
-	reagent_transfer_amount = 8
+	smoke_strength = 0.75
+	reagent_transfer_amount = 8.5
 
 /datum/ammo/xeno/toxin/medium/upgrade2
-	smoke_strength = 0.7
-	reagent_transfer_amount = 8.5
+	smoke_strength = 0.8
+	reagent_transfer_amount = 9
 
 /datum/ammo/xeno/toxin/medium/upgrade3
-	smoke_strength = 0.75
-	reagent_transfer_amount = 9
+	smoke_strength = 0.85
+	reagent_transfer_amount = 9.5
 
 
 /datum/ammo/xeno/toxin/heavy //Praetorian
 	name = "neurotoxic splash"
-	added_spit_delay = 15
-	spit_cost = 100
-	damage = 40
-	smoke_strength = 0.65
-	reagent_transfer_amount = 8.5
+	added_spit_delay = 13
+	spit_cost = 120
+	damage = 50
+	smoke_strength = 0.75
+	reagent_transfer_amount = 9
 
 /datum/ammo/xeno/toxin/heavy/upgrade1
-	smoke_strength = 0.7
-	reagent_transfer_amount = 9
+	smoke_strength = 0.8
+	reagent_transfer_amount = 10
 
 /datum/ammo/xeno/toxin/heavy/upgrade2
-	smoke_strength = 0.75
-	reagent_transfer_amount = 9.5
+	smoke_strength = 0.9
+	reagent_transfer_amount = 10.5
 
 /datum/ammo/xeno/toxin/heavy/upgrade3
-	smoke_strength = 0.8
-	reagent_transfer_amount = 10
+	smoke_strength = 0.10
```

Comment:
probably should have this be 1.0 instead of 0.10

Reply:
^^